src/main/java/com/everest/employeeportal/entities/Employee.java

```diff
@@ -2,20 +2,46 @@
 
 import lombok.Data;
 
+
+import javax.persistence.*;
 import java.time.LocalDate;
+import java.util.Set;
 
 @Data
+@Entity
+@Table(name = "employees")
 public class Employee {
-    private Long employeeId;
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @Column(name = "firstName")
     private String firstName;
+
+    @Column(name = "lastName")
     private String lastName;
+
+    @Column(name = "password")
     private String password;
+
+    @Column(name = "personalEmail")
     private String personalEmail;
+
+    @Column(name = "everestEmail")
     private String everestEmail;
+
+    @Column(name = "DOB")
     private LocalDate dateOfBirth;
+
+    @Column(name = "DOJ")
     private LocalDate dateOfJoining;
+
+    @Column(name = "Experience")
     private Integer yearsOfExperience;
+
+    @Column(name = "Bio")
     private String bio;
-    private Address currentAddress;
-    private Address permanentAddress;
+
+    @OneToMany(mappedBy = "employee", fetch = FetchType.LAZY, cascade = CascadeType.ALL)
+    private Set<Address> currentAddress;
```

Comment:
How do you identify which is present address and permanent address?

Reply:
Added present and permanent address in fix address relations PR.